### PR TITLE
DP-2706 Adding enforcing api token related APIs to documentation

### PR DESCRIPTION
**Pull request by JKlucka**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

File: docs/content/en/docs/administration/permissions/manage_organization_permissions.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+weight: 20
+no_list: true
+superheading: "catalog_permission."
+---
+
+
+
+``manage_organization_permissions(organization_permission_assignments: List[CatalogOrganizationPermissionAssignment])``
+
+Manage organization the permissions.
```
Comment (Mara3l) [inline docs/content/en/docs/administration/permissions/manage_organization_permissions.md:13]:
```suggestion
Manage the permissions of the whole organization.
```

File: docs/content/en/docs/administration/permissions/get_declarative_organization_permissions.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+superheading: "catalog_permission."
+---
+
+
+
+``get_declarative_organization_permissions()``
+
+Gets the current set of permissions of the workspace in a declarative form.
+
+{{% parameters-block  title="Parameters"%}}
```
Comment (Mara3l) [inline docs/content/en/docs/administration/permissions/get_declarative_organization_permissions.md:15]:
```suggestion
{{% parameters-block  title="Parameters" None="yes" %}}
```

File: docs/content/en/docs/administration/permissions/put_declarative_organziation_permissions.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+weight: 20
+no_list: true
+superheading: "catalog_permission."
+---
+
+
+
+``put_declarative_organization_permissions(org_permissions: CatalogDeclarativeOrganizationPermission)``
+
+Sets the permissions for the organziation.
```
Comment (Mara3l) [inline docs/content/en/docs/administration/permissions/put_declarative_organziation_permissions.md:13]:
```suggestion
Sets the permissions for the whole organization.
```

File: docs/content/en/docs/administration/permissions/manage_organization_permissions.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+{{% parameters-block  title="Parameters"%}}
+{{< parameter p_name="organization_permission_assignments" p_type="List[CatalogOrganizationPermissionAssignment]" >}}
+List of Object Containing Organization Permissions Assignments.
```
Comment (Mara3l) [inline docs/content/en/docs/administration/permissions/manage_organization_permissions.md:17]:
```suggestion
List of Organization Permission Assignments.
```

File: docs/content/en/docs/administration/permissions/put_declarative_organziation_permissions.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+{{% parameters-block  title="Parameters"%}}
+{{< parameter p_name="org_permissions" p_type="List[CatalogDeclarativeOrganizationPermission]" >}}
+List of Objects Containing Organization Permissions.
```
Comment (Mara3l) [inline docs/content/en/docs/administration/permissions/put_declarative_organziation_permissions.md:17]:
```suggestion
List of Organization Permissions.
```

**Review by Mara3l — CHANGES_REQUESTED** [review]
Please update the markdown files. Also please update the `_index.md` in the same directory and add all of these methods to the list at the line 15

File: docs/content/en/docs/administration/permissions/get_declarative_organization_permissions.md (hunk 2 above)
Comment (JKlucka) [reply docs/content/en/docs/administration/permissions/get_declarative_organization_permissions.md:15]: ok, thanks

File: docs/content/en/docs/administration/permissions/manage_organization_permissions.md (hunk 1 above)
Comment (JKlucka) [reply docs/content/en/docs/administration/permissions/manage_organization_permissions.md:13]: 👍

File: docs/content/en/docs/administration/permissions/manage_organization_permissions.md (hunk 4 above)
Comment (JKlucka) [reply docs/content/en/docs/administration/permissions/manage_organization_permissions.md:17]: 👍

File: docs/content/en/docs/administration/permissions/put_declarative_organziation_permissions.md (hunk 3 above)
Comment (JKlucka) [reply docs/content/en/docs/administration/permissions/put_declarative_organziation_permissions.md:13]: 👍

File: docs/content/en/docs/administration/permissions/put_declarative_organziation_permissions.md (hunk 5 above)
Comment (JKlucka) [reply docs/content/en/docs/administration/permissions/put_declarative_organziation_permissions.md:17]: 👍

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by JKlucka** [comment]
@Mara3l thanks for the review. updated as recommended. May I ask for second round, please? Thank you

File: docs/content/en/docs/administration/permissions/put_declarative_organziation_permissions.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
+---
```
Comment (Mara3l) [inline docs/content/en/docs/administration/permissions/put_declarative_organziation_permissions.md:1]: There is a typo in the name of the file, please update to `put_declarative_organization_permissions`

**Review by Mara3l — CHANGES_REQUESTED** [review]

File: docs/content/en/docs/administration/permissions/put_declarative_organziation_permissions.md (hunk 6 above)
Comment (Mara3l) [reply docs/content/en/docs/administration/permissions/put_declarative_organziation_permissions.md:1]: @JKlucka This would be the last change :)

Comment (JKlucka) [reply docs/content/en/docs/administration/permissions/put_declarative_organziation_permissions.md:1]: Omg, sorry for that! Thank you

Comment (JKlucka) [reply docs/content/en/docs/administration/permissions/put_declarative_organziation_permissions.md]: Done @Mara3l

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Review by Mara3l — APPROVED** [review]

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-python-sdk-gate-PR402

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Pull request merged**